buildSrc/src/main/java/dependencies/Dependencies.kt

```diff
@@ -22,7 +22,7 @@ object Dependencies {
     const val androidx_preference_ktx = "androidx.preference:preference-ktx:1.1.1"
     const val androidx_fragment_ktx = "androidx.fragment:fragment-ktx:${Versions.androidx_fragment}"
     const val android_material = "com.google.android.material:material:1.4.0"
-    const val android_flexbox = "com.google.android:flexbox:2.0.1"
+    const val android_flexbox = "com.google.android.flexbox:flexbox:3.0.0"
```

Comment:
Is this necessary? If not I would rather keep the same version since we are about to release a new version and this pr is not for things like that.

Reply:
3.0.0 is the only version available in the Google maven repo annoyingly: https://maven.google.com/web/index.html#com.google.android.flexbox:flexbox. It looked like the upgrade was mostly around build tooling and the move out of JCenter from the [release notes](https://github.com/google/flexbox-layout/releases/tag/3.0.0).

Reply:
Ah ok